pkg/virt-controller/services/renderresources_test.go

```diff
@@ -444,6 +444,40 @@ var _ = Describe("Resource pod spec renderer", func() {
 	)
 })
 
+var _ = Describe("Tolerations pod spec renderer", func() {
+	DescribeTable("Tolerations for hotplug pod", func(kvConfig v1.KubeVirtConfiguration, expectedTolerations []kubev1.Toleration) {
+		Expect(hotplugPodTolerations()).To(BeEquivalentTo(expectedTolerations))
+	},
+		Entry("empty tolerations, fallback to default", v1.KubeVirtConfiguration{}, []kubev1.Toleration{
```

Comment:
Do we expect to add more entries here in the near future? If not, this should just be an `It` and not a single-entry `DescribeTable`.